samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs

```diff
@@ -82,7 +86,7 @@ public class SemanticKernelController : ControllerBase
         // Register skills with the planner if enabled.
         if (plannerOptions.Value.Enabled)
         {
-            await this.RegisterPlannerSkillsAsync(planner, plannerOptions.Value);
+            await this.RegisterPlannerSkillsAsync(planner, plannerOptions.Value, openApiSkillsAuthHeaders);
```

Comment:
Just curious and would like to learn more. Why we need to set request header?

Reply:
Calls to some open apis (github and jira) will fail without some form of oauth. The headers at least tell us there is some auth combo that the user is trying. Since the openAPI plugins will fail without Oauth it doesnt make sense to expose them as skills to the planner/kernel since it wont be able to use them in its planning anyway.

Reply:
@Vivihung + we need to isolate separate auth headers for each plug-ins so the server knows how to map the auth data to the respective plug-ins. If the auth data isn't available, the skill won't be registered